Assets/Scripts/Spawner.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Spawner : MonoBehaviour
+{
+    private List <List <int>> waves;
+    public GameObject[] enemies; 
+    public GameObject waypoints;
+    public int timeBetweenWaves;
+    public int timeBetweenSpawns;
+    private float timer;
+
+    private bool spawningWave;
+    private int waveIndex;
+    private int enemyIndex;
+
+    private void Start(){
+        timer = timeBetweenWaves;
+        spawningWave = false;
+        waveIndex = 0;
+        enemyIndex = 0;
+        List<int> wave1 = new List<int>{0,0,0};
+        List<int> wave2 = new List<int>{0,0,0,0,0};
+        List<int> wave3 = new List<int>{0,0};
+        List<int> wave4 = new List<int>{0,0,0,0,0};
```

Comment:
Do you think it would be better to make a wave object or something that allows it to be a little simple to add, edit, and change waves?

Reply:
Will be done next sprint.